python/ray/ray_constants.py

```diff
@@ -32,6 +32,9 @@ def env_bool(key, default):
 DEFAULT_REDIS_MAX_MEMORY_BYTES = 10**10
 # The smallest cap on the memory used by Redis that we allow.
 REDIS_MINIMUM_MEMORY_BYTES = 10**7
+# If a user does not specify a port for the primary Ray service,
+# we attempt to start the service running at this port.
+DEFAULT_PORT = 52600
```

Comment:
We should match this to the current defaults we use in the autoscaler configs, 6379:
https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/example-full.yaml#L137